tf/train.py

```diff
@@ -418,6 +418,10 @@ def main(cmd):
     experimental_reads = max(2, mp.cpu_count() - 2) // 2
     extractor = select_extractor(tfprocess.INPUT_MODE)
 
+    if experimental_parser and (not value_focus_min == 1 or not value_focus_slope == 0):
+            raise ValueError('Experimental parser does not support non-default value \
```

Comment:
This formatting looks a bit strange.
Also using not blah == 1 instead of blah != 1 feels wrong to me, but I'm not a python expert :P

Reply:
Still looks like excessive indentation on the next line.

Reply:
oh okay i guess